config/cable.yml

```diff
@@ -0,0 +1,9 @@
+development:
+  adapter: async
+
+test:
+  adapter: async
+
+production:
+  adapter: redis
+  url: redis://localhost:6379/1
```

Comment:
This should be an ENV variable.

Reply:
I can't find documentation about how to replace the yaml config with environment variables. (My assumption is that, since this isn't a .rb file, it isn't executable, so I can't just write something like `url: ENV['REDIS_URL']`) Can you point me to the right direction?

Reply:
Thanks